lib/errors.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = (translate, options) => {
+
+  var logger = (options || {}).logger;
+
+  if (typeof translate !== 'function') {
+    throw new Error('Errorhandler needs a translate function');
+  }
+
+  return (err, req, res) => {
```

Comment:
I just noticed there is no `next` param. 
![](
![image](https://cloud.githubusercontent.com/assets/1274835/14981038/5efccfda-1124-11e6-9617-5fd464f3e4d7.png))